.github/pull_request_template.md

```diff
@@ -9,16 +9,49 @@
 
 ## Issue type checklist
 <!--- What type of change are you submitting? Check the boxes that apply: -->
-- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
-- [ ] **New feature**: non-breaking change that adds functionality.
-- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
+
+- [] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
+- [] **New feature**: non-breaking change that adds functionality.
+- [] **Breaking change**: fix or feature that will cause current functionality to change.
 
 ## Process & policy checklist
 <!--- Review the list and check the boxes that apply. -->
-- [ ] I have added tests for my changes.
-- [ ] I have tested my changes.
-- [ ] I have updated the project documentation to reflect my changes.
-- [ ] I have read the CONTRIBUTING documentation and followed the standards set for this project.
 
- 
-Click here for details on [formatting pull requests](https://microsoft.github.io/fast-dna/docs/contributing/working#formatting-messages).
+- [] I have added tests for my changes.
+- [] I have tested my changes.
+- [] I have updated the project documentation to reflect my changes.
+- [] I have read the CONTRIBUTING documentation and followed the standards set for this project.
+
+## Pull request formatting
+
+**`<type>`** is required to be at least one of the following:
+
+- break: update that changes the current functionality
+- chore: update that does not impact distributed packages
+- feat: add, update, or remove a feature
+- fix: remove property to resolve style issue
+
+**`<description>`** is required and speaks to what the user gets from this PR:
+
+- Be concise
+- Use lowercase
+- Use imperative, present tense (e.g. `add` not `adds`)
+- Do not end with a period. Period.
+- Avoid redundant words
+- Start your description with `add`, `update`, `remove`, or `break`
+
+Accepted title format
+
+```comment
+<type>: <description>
+```
+
+Example titles
+
+```comment
+chore: add unit tests for all components
+feat: add a border radius to button
+fix: update design system to use 3px border radius
+```
+
+For additional details working on FAST-DNA check out our [documentation site](https://microsoft.github.io/fast-dna/docs/en/contributing/working).
```

Comment:
Comma after FAST-DNA:
"... on FAST-DNA, check out ..."

I think you also need an extra word, something like "For additional details **regarding** working on".